library/certs.c

```diff
@@ -40,24 +40,24 @@
 /* BEGIN FILE string macro TEST_CA_CRT_EC_PEM tests/data_files/test-ca2.crt */
```

Comment:
Please update the files in `tests/data_files` to match. Otherwise it's going to be confusing that they don't match, and it will make the CI fail if we finish the work to generate `certs.c` automatically.